pages/data-design/start-using-inspector.mdx

```diff
@@ -1,16 +1,22 @@
 ---
 layout: 'guide'
-title: 'Getting started to use Inspector to help identify issues to fix in your legacy events'
+title: 'Use Inspector to improve your tracking plan health'
 ---
 
 import Link from '../../components/Link';
 
-After setting up your sources to send data to the Avo Inspector with the <Link passHref href="/implementation/avo-inspector-sdk-reference"><a>Avo Inspector SDKs</a></Link> you will see the data appearing in the <Link passHref href="/workspace/inspector"><a>the Inspector tab</a></Link> tab.
+After setting up your sources to send data to the Avo Inspector with the <Link passHref href="/implementation/avo-inspector-sdk-reference"><a>Avo Inspector SDKs</a></Link> you will see the data appearing in the the Inspect tab.
 
-Open the <Link passHref href="/workspace/inspector#default-view-and-switching-between-views">Inspector events screen view</Link>, browse the list or search for specific event, select it and the right panel will appear with detailed information about the health of the given event.
+<Link passHref href="/workspace/inspector">Inspect tab description</Link>
+<p/>
```

Comment:
Should this closing p be here?

Reply:
Yeah, I'm not sure why double space does not create a new line 🤔 But the way to solve this should be <br/>, not <p/>